docs/sensors/filters/data.md

```diff
@@ -0,0 +1,206 @@
+
+# Data Filter
+
+Data filters are applied to the event data. A CloudEvent from Webhook event-source has payload structure as:
+
+```json
+{
+    "context": {
+      "type": "type_of_event_source",
+      "specversion": "cloud_events_version",
+      "source": "name_of_the_event_source",
+      "id": "unique_event_id",
+      "time": "event_time",
+      "datacontenttype": "type_of_data",
+      "subject": "name_of_the_configuration_within_event_source"
+    },
+    "data": {
+      "header": {},
+      "body": {},
+    }
+}
+```
+
+Data filters are applied on `data` within the payload.
+
+## Fields
+
+A data filter has following fields:
+
+```yaml
+filters:
+  dataLogicalOperator: logical_operator_applied
+  data:
+    - path: path_within_event_data
+      type: types_of_the_data
+      comparator: numeric_comparator
+      value:
+        - list_of_possible_values
+```
+
+> ⚠️ `PLEASE NOTE` order in which data filters are declared corresponds to the order in which the Sensor will evaluate them.
+
+## Logical operator
+
+Data filters can be evaluated together in 2 ways:
+
+- `AND`, meaning that all data filters returning `true` are required for an event to be valid
```

Comment:
Does uppercase/lowercase matter? It's lowercase in the example, should be consistent.

Reply:
actually yeah matters, so I will add this point to the doc :) thanks!

Reply:
fixed